src/application/tactics/spin_on_init.c

```diff
@@ -0,0 +1,93 @@
+/*!
+ * @file
+ * @brief
+ */
+
+#include "spin_on_init.h"
+#include "tactic.h"
+#include "tiny_utils.h"
+#include "tuning.h"
+#include "i_tiny_adc.h"
+#include <stdio.h>
+
+enum {
+  max_counts = 0xFFFF
```

Comment:
```suggestion
  max_counts = UINT16_MAX
```

Reply:
idk why I couldn't remember what the macro was